hedera-node/src/main/java/com/hedera/services/state/logic/StandardProcessLogic.java

```diff
@@ -83,6 +84,14 @@ public StandardProcessLogic(
     public void incorporateConsensusTxn(
             Transaction platformTxn, Instant consensusTime, long submittingMember) {
         try {
+            // Deduct 1000 nanos from the consensusTime allotted by platform, to accommodate the
+            // preceding,
+            // following child records and any long term scheduled transactions triggered by the
+            // current transaction
+            // in the balance file with consensus timestamp X to include all transactions whose
+            // consensus time T <= X.
+            consensusTime = consensusTime.minusNanos(EventImpl.MIN_TRANS_TIMESTAMP_INCR_NANOS);
```

Comment:
I worry that this might break something that depends on the platform timestamp being used by at least one txn, but I can't think of anything. This should work.

Should this use `com.hedera.services.utils.Units.MIN_TRANS_TIMESTAMP_INCR_NANOS` instead of `EventImpl.MIN_TRANS_TIMESTAMP_INCR_NANOS`?

Reply:
Thanks Patrick. Changed to use from `Units` instead.